### ISSUE TITLE
old_div fixer misses some expressions

### ISSUE THREAD

**Issue by stuarteberg** [PythonCharmers/python-future#263]
I'm confused by the behavior of the `futurize` script's `old_div` fixer.  Given the file below, I'm wondering: 

- Why does the fixer skip the second case?
- Why *doesn't* the fixer skip the third case?

(Sorry if I'm missing something -- I'm experienced with Python 2, but I'm a Python 3 novice.)

```python
# This line gets fixed
a1 = b / c

# But this line gets skipped?
a2 = 1*b / c

# Shouldn't this be skipped?
a3 = 1.0 / c
```

```diff
$ futurize foo.py
RefactoringTool: Skipping optional fixer: idioms
RefactoringTool: Skipping optional fixer: ws_comma
RefactoringTool: Refactored foo.py
--- foo.py	(original)
+++ foo.py	(refactored)
@@ -1,8 +1,10 @@
+from __future__ import division
 # This line gets fixed
-a1 = b / c
+from past.utils import old_div
+a1 = old_div(b, c)

 # But this line gets skipped?
 a2 = 1*b / c

 # Shouldn't this be skipped?
-a3 = 1.0 / c
+a3 = old_div(1.0, c)
RefactoringTool: Files that need to be modified:
RefactoringTool: foo.py
```

**Comment by r3m0t** [comment]
Works in current version

**Comment by jmadler** [comment]
These were limitations in the fixer that were addressed in 0.17.0 and the upcoming 0.18.0
